<?xml version="1.0"?>
<launch>

  <!--
    Launch node for subscribing Odometry messages and publishing them as tf2 messages
  -->
  <arg name="odo_topic"         default="/pose" />
  <arg name="time_now"          default="false"/>
  <arg name="odo_queue_size"    default="1"/>


  <node pkg="tf_server" type="odo_to_tf2_node" name="odo_to_tf2" output="screen">
    <param name="odo_topic"       value="$(arg odo_topic)"/>
    <param name="time_now"        value="$(arg time_now)"/>
    <param name="odo_queue_size"  value="$(arg odo_queue_size)"/>
  </node>

</launch>
